### DISCUSSION TITLE
Include "Generic context" in webhook payload

### DISCUSSION THREAD

**Discussion by AndreVME** [spatie/flareapp.io-roadmap#57] [Ideas]
User Story: As a developer I would like to receive the log's contextual information within flare's webhook payload so that the application receiving that webhook can display pertinent information to aid me with debugging. 

The Laravel logger allows for [contextual information](https://laravel.com/docs/9.x/logging#contextual-information) to be added. This information is made available in the flare dashboard within the "context" tab. Would it also be able to output this information within the payload of the webhook notifications?

**Comment by AlexVanderbist** [comment]
We have no plans to add context to the webhook payload. Our goal is to keep the webhooks lightweight. If you require more context data for webhook errors, you can utilize the API to retrieve the complete error occurrence and its associated context.
